global.json

```diff
@@ -1,5 +1,5 @@
 {
     "sdk": {
-        "version": "2.1.4"
+        "version": "2.1.500"
```

Comment:
Why 500 here?

Reply:
@ffMathy oh that's just from my initial setup cause I don't have 2.1.4 locally and when I did that I was just experimenting, didn't consider I'd end up making a PR...

Reply:
Ah I see.